tensorflow/c/eager/BUILD

```diff
@@ -416,9 +416,7 @@ tf_cuda_cc_test(
     ],
     args = ["--heap_check=local"],
     linkstatic = tf_kernel_tests_linkstatic(),
-    tags = tf_cuda_tests_tags() + [
```

Comment:
Let's leave the `no_cuda_asan` in the BUILD targets for now unless you have been able to test that the leaks are gone. I can try to remove this and test locally once your changes are merged. It might take longer to merge this if the tests are still failing since we will have to wait for the internal tests, add this back, re-pull etc.

Reply:
Done